backend/app/components/spree/admin/pill_component.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Spree::Admin::PillComponent < ActionView::Component::Base
+  STATES = %i[
```

Comment:
This should be `STATUS` to avoid confusion with `Spree::State` (part of Country) and the formal plural of status is status (https://english.stackexchange.com/questions/877/what-is-the-plural-form-of-status)